pkg/lib/docker/errors.go

```diff
@@ -57,10 +57,18 @@ func ErrorDockerPermissions(err error) error {
 }
 
 func ErrorImageInaccessible(image string, cause error) error {
-	dockerErrMsg := errors.Message(cause)
+	dockerErrMsg := ""
```

Comment:
Looks cleaner, I agree. Shouldn't we place the message before the docker error message? The latter is a lot more verbose.

```go
message := fmt.Sprintf("%s is not accessible", image)
if cause != nil {
	message += "\n" + errors.Message(cause) // add \n because docker client errors are
}
```

Reply:
Yes I think that make sense (and that's the way it was before too)